helm_deploy/values-dev.yaml

```diff
@@ -27,7 +27,9 @@ generic-service:
    CHECK_MY_DIARY_URL: https://check-my-diary-dev.prison.service.justice.gov.uk?fromDPS=true
    INCENTIVES_URL: https://incentives-ui-dev.hmpps.service.justice.gov.uk
    INCENTIVES_API_ENDPOINT_URL: https://incentives-api-dev.hmpps.service.justice.gov.uk
+   NON_ASSOCIATIONS_PRISONS: "MDI"
```

Comment:
can we have 1 or 2 more? so that it's easier to test journeys with the "in other establishments" section on our site

Reply:
Good point, I added a few more in this commit: https://github.com/ministryofjustice/digital-prison-services/pull/2554/commits/5d3488ac28779987b7c77b9062177890d701b46d